### ISSUE TITLE
Allow output logs to CloudWatch for aws_ssm_maintenance_window_task resource

### ISSUE THREAD

**Issue by ispirals** [hashicorp/terraform-provider-aws#14136]
### Community Note

* Please vote on this issue by adding a 👍 [reaction](https://blog.github.com/2016-03-10-add-reactions-to-pull-requests-issues-and-comments/) to the original issue to help the community and maintainers prioritize this request
* Please do not leave "+1" or other comments that do not add relevant new information or questions, they generate extra noise for issue followers and do not help prioritize the request
* If you are interested in working on this issue or have submitted a pull request, please leave a comment

### Description

In AWS a system manager maintenance window task can define two different modes for the output of the commands executed in the task: S3 or CloudWatch. 

The AWS CLI allows this functionallity with the parameter `--task-invocation-parameters` in the `register-task-with-maintenance-window` action.

```json
...
"CloudWatchOutputConfig": {
  "CloudWatchLogGroupName": "string",
  "CloudWatchOutputEnabled": true|false
},
...
```

The terraform provider only allows the s3 bucket options under the `run_command_parameters` argument.

My proposal is to add a new argument `cloudwatch_log_group_name` under the `run_command_parameters` argument.

### New or Affected Resource(s)

* aws_ssm_maintenance_window_task

### Potential Terraform Configuration

Example of the proposal
```hcl
resource "aws_ssm_maintenance_window_task" "run_patch_baseline" {
  max_concurrency  = 2
  max_errors       = 1
  priority         = 2
  service_role_arn = aws_iam_role.aws_role_ssm.arn
  task_arn         = "AWS-RunPatchBaseline"
  task_type        = "RUN_COMMAND"
  window_id        = aws_ssm_maintenance_window.window.id

  targets {
    key    = "WindowTargetIds"
    values = [aws_ssm_maintenance_window_target.targets.id]
  }

  task_invocation_parameters {
    run_command_parameters {
      cloudwatch_log_group_name = "ssm-log-group"
  
      parameter {
        name   = "Operation"
        values = ["Install"]
      }
    }
  }
}
```

### References

* https://docs.aws.amazon.com/cli/latest/reference/ssm/register-task-with-maintenance-window.html

**Comment by sj158** [comment]
Same for "aws_ssm_document"

**Comment by brendan-sherrin** [comment]
Use case: We're patching using SSM and I'm trying to arrange maintenance windows in our monitoring system based on when patching starts and finishes.  Sending these logs to Cloudwatch would assist in this.

**Comment by red8888** [comment]
any updates on this? I dont want to write to an an s3 bucket just to a cloudwatch log group

**Comment by bflad** [comment]
Support for this functionality has been merged and will release with version 3.28.0 of the Terraform AWS Provider, later today. Thank you to @DrFaust92, @0x91, and @mycodeself for the implementation efforts. 👍
